.github/workflows/update_app_templates_hash.yml

```diff
@@ -0,0 +1,43 @@
+name: Update aws/aws-sam-cli with latest commit hash from aws/aws-sam-cli-app-templates
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+  workflow_dispatch: {}
+
+jobs:
+  updateCommitHash:
+    permissions:
+      pull-requests: write
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout App Templates
```

Comment:
Do we need to specify a branch?

Reply:
We are cloning default branch in SAM CLI and here as well. So we don't need to maintain a branch configuration here.